Watchman.DomainModel/Settings/ConfigurationItems/TrustedUserRolesNames.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+
+namespace Watchman.DomainModel.Settings.ConfigurationItems
+{
+    public class TrustedUserRolesNames : MappedConfiguration<IEnumerable<string>>
```

Comment:
czemu to jest w konfguracji?

Reply:
W sumie z przyzwyczajenia tego użyłem, ale mogę zrobić jako normalną osobną kolekcję